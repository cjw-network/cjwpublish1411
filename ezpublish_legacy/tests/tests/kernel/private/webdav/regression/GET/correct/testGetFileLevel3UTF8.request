<?php

// Test if GET returns correct information for a file at $GLOBALS['ezc_webdav_testfolder']
// with UTF-8 characters in the name

// First upload the file as in the folder $GLOBALS['ezc_webdav_testfolder']
$article = new ezpObject( 'file', (int)$GLOBALS['ezc_webdav_testfolderid'] );
$article->name = "nørsk deütsch русский";
$article->file = dirname( __FILE__ ) . DIRECTORY_SEPARATOR . $GLOBALS['ezc_webdav_testfolder'] . '.txt';
$article->publish();

$_SERVER['REQUEST_URI'] = '/' . $GLOBALS['ezc_siteaccess'] . '/Content/' . $GLOBALS['ezc_webdav_testfolder'] . '/' . urlencode( "nørsk deütsch русский.txt" );
$_SERVER['REQUEST_METHOD'] = 'GET';

?>
