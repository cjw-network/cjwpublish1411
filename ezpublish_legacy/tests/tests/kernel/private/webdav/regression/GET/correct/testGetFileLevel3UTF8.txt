PHP Requirements
================

As PHP version the eZ Components require version **5.2.1**, although a later
version is always recommended. The latest version of PHP can be downloaded
here__.

eZ Components versions before the 2008.1 release require PHP version
**5.1.6**. Please be aware that no updates for those releases are being made
anymore.