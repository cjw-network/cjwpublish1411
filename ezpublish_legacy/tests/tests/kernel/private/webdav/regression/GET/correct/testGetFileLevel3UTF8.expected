ETag: 28e11455fd37b93aba4d8367e9602baa
Server: eZComponents/dev/ezcWebdavTransportWrapper
Content-Type: text/plain; charset="utf-8"
Content-Length: 14903
HTTP/1.1 200 OK
